bindings/boost_python/class.cpp.tmpl

```diff
@@ -48,7 +51,7 @@ void {{class.mangled_name}}()
 {{#class.constructors}}{{!
 }}{{#overloads}}{{!
     }}.def("__init__", ::boost::python::make_constructor({{!
-        }}[]({{#params}}{{type}} {{name}}{{^last}}, {{/last}}{{/params}}){{!
+        }}+[]({{#params}}{{type}} {{name}}{{^last}}, {{/last}}{{/params}}){{!
```

Comment:
Adding this '+' operator triggers a conversion of the lambda to a plain function pointer, which resolves compilation issues in boost::python not being able to deduce the type of the lambda function.

See:
http://stackoverflow.com/questions/16845547/using-c11-lambda-as-accessor-function-in-boostpythons-add-property-get-sig
http://stackoverflow.com/questions/18889028/a-positive-lambda-what-sorcery-is-this